pygenometracks/tests/test_data/browser_tracks_hic_force_scale.ini

```diff
@@ -0,0 +1,27 @@
+
+[hic matrix]
+file = Li_et_al_2015.h5
+title = depth = 500000; colormap = PuRd; min_value = 5;
+        max_value = 70
+min_value = 5
+max_value = 70
+depth = 500000
+colormap = PuRd
+file_type = hic_matrix
+show_masked_bins = false
+
+[spacer]
+height = 0.5
+
+[hic matrix]
+file = Li_et_al_2015.h5
+title = depth = 1000000; colormap = PuRd; min_value = 0;
+        max_value = 80
```

Comment:
Is there a reason for the additional space?

Reply:
It is to make it part of the title not a new parameter but I can put all in the same line if it is easier to understand.

Reply:
I think it would be better to write it in quotation marks. 

```
title = 'depth = 500000; colormap = PuRd; min_value = 5;
        max_value = 70'
```

Currently it is not that obvious for users where the difference of this max_value is with e.g. the one in line 21. I fear if we leave it this way many will be confused.

Reply:
If I put the quote or the `\n`:
![image](https://user-images.githubusercontent.com/30404086/97973278-223bb300-1dc6-11eb-9de8-c9e3ab7e3a89.png)
I get them in the title:
![image](https://user-images.githubusercontent.com/30404086/97973340-35e71980-1dc6-11eb-9ed9-3a5fee1c2d40.png)
The only way to go back to line is to use indentation: 
https://docs.python.org/3.4/library/configparser.html?highlight=configfile#supported-ini-file-structure
"Values can also span multiple lines, as long as they are indented deeper than the first line of the value."
But for clarity I will put all titles in only one line.